MNIST/convnet.py

```diff
@@ -143,6 +143,25 @@ def initialize_param(f, l):
 def initialize_theta(NUM_OUTPUT, l_in):
 	return 0.01*np.random.rand(NUM_OUTPUT, l_in)
 
+def initialise_param_lecun_normal(FILTER_SIZE, IMG_DEPTH, scale=1.0, distribution='normal'):
+	
+    if scale <= 0.:
+            raise ValueError('`scale` must be a positive float. Got:', scale)
+
+    distribution = distribution.lower()
+    if distribution not in {'normal'}:
+        raise ValueError('Invalid `distribution` argument: '
+                             'expected one of {"normal", "uniform"} '
+                             'but got', distribution)
+
+    scale = scale
+    distribution = distribution
+    fan_in = FILTER_SIZE*FILTER_SIZE*IMG_DEPTH
+    scale /= max(1., fan_in)
+    stddev = scale * np.sqrt(1./fan_in)
```

Comment:
As far as i have read about lecun_normal, It should have stddev of about np.sqrt(1./fan_in) & not (1./fan_in)*np.sqrt(1./fan_in) [given  by line no. 161 --  scale*np.sqrt(1./fan_in) where scale=(1./fan_in)]  which is in this case.